view/templates/field_yesno.tpl

```diff
@@ -1,16 +0,0 @@
-
-	<div class="field yesno">
```

Comment:
Please check none of the removed classes are specific to this template and will leave orphan CSS.